docs/application/dotnet/get-started/wearable/first-xaml-app.md

```diff
@@ -126,6 +130,30 @@ If you face any issues during deployment, it is recommended to manually install
 ```bash
 $ sdb install <path-to-package>/org.tizen.example.TizenWearableXamlApp1-1.0.0.tpk
 ```
+## Validate Your Application Changes Using XAML Hot Reload
```

Comment:
Sorry for the missing the title checking.

As we know, we are going to use sentence-style capitalization.

However to get the consistency, in this page it is better to use capital letter for the every 1st letters, as like other titles.
(I think the titles will be updated in a lump in the future.)

But... Does the "Using" also require capital letter? Would you please let me know your opinion?

Reply:
Will using lower case for the word using.